project_resource_calendar/models/instrument.py

```diff
@@ -27,3 +27,15 @@ class Instrument(models.Model):
         index=True,
         default='instrument',
     )
+    instrument_log_count = fields.Integer(
+        string='Calendar Logs',
+        compute='_compute_instrument_log_count',
+    )
+
+    def _compute_instrument_log_count(self):
```

Comment:
Add tests for this method, we agreed on adding tests for all new code this week

Reply:
@eilst  Yes doing in the different branch

Reply:
@eilst Doing in the different branch